src/main/kotlin/org/rust/lang/core/completion/RsDeriveCompletionProvider.kt

```diff
@@ -57,15 +69,48 @@ object RsDeriveCompletionProvider : RsCompletionProvider() {
                 // and 'traitWithDependencies' contains only 'derivable' element
                 if (traitWithDependencies.size > 1) {
                     val element = LookupElementBuilder.create(traitWithDependencies.joinToString(", "))
-                        .withIcon(RsIcons.TRAIT.multiple())
+                        .withIcon(RsIcons.MACRO)
```

Comment:
@dima74 it's quite unexpected change in this PR, i.e. that we change icon of items not related to PR, dropping `.multiple()` and not mentioned it in description at all. Moreover, gif in the PR description shows wrong icons.

I understand that `RsIcons.MACRO.multiple()` looks bad but `RsIcons.MACRO` looks quite unexpected for me in general in `derive` attribute (and especially for builtin derived traits) because `RsIcons.MACRO` and `RsIcons.MACRO2` were created for bang macros. That's why they contains `!` symbols. More precisely,  `RsIcons.MACRO` is supposed to be used only with `macro_rules!` macros and `RsIcons.MACRO2` for macro 2.0 (and probably for procedural bang macros)

Reply:
I agree that chosen icons are not ideal, I changed them for built-in derives for consistency. I guess we need new icons for attribute and derive proc macros

Reply:
To be more clear - I like this completion improvement very much 🔥 
Just want not to introduce confusions for users and other plugin developers/QA. Thanks for updating PR description!